Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/PressableButtons/PressableButton.cs

```diff
@@ -257,44 +190,32 @@ void IMixedRealityTouchHandler.OnTouchStarted(HandTrackingInputEventData eventDa
         void IMixedRealityTouchHandler.OnTouchUpdated(HandTrackingInputEventData eventData)
         {
             touchPoints.Add(eventData.InputData);
-            eventData.Use();
         }
         
         void IMixedRealityTouchHandler.OnTouchCompleted(HandTrackingInputEventData eventData)
         {
-            eventData.Use();
         }
         
         #endregion OnTouch
 
         #region private Methods
 
-        private void SetPathMarkers()
+        public void FindOrCreatePathMarkers()
         {
-            GameObject initialMarker = new GameObject("Initial");
-            initialMarker.hideFlags = HideFlags.HideInHierarchy | HideFlags.HideInInspector;
+            Transform sourcePositionTransform = (movingButtonVisuals != null) ? movingButtonVisuals.transform : transform;
 
-            if (movingButtonVisuals != null)
-            {
-                initialMarker.transform.position = movingButtonVisuals.transform.position;
-                initialMarker.transform.parent = movingButtonVisuals.transform.parent;
-            }
-            else
+            // First try to search for our markers
+            if (initialTransform == null)
             {
-                initialMarker.transform.position = transform.position;
-                initialMarker.transform.parent = transform.parent;
+                initialTransform = transform.Find(InitialMarkerTransformName);
             }
 
-            initialTransform = initialMarker.transform;
-        }
-
-        private void ClearPathMarkers()
-        {
-            if (initialTransform != null)
+            // If we don't find them, create them
+            if (initialTransform == null)
             {
-                initialTransform.parent = null;
-                DestroyImmediate(initialTransform.gameObject);
-                initialTransform = null;
+                initialTransform = new GameObject(InitialMarkerTransformName).transform;
```

Comment:
Do we even need to create a new GameObject and save this as a transform? As far as I can tell, we only use its position. Could we just save off the `Vector3` position?

Reply:
For now we need at least a transform in order for correctly pushing the button in local space. This way the button can have a parent node that is moving (like a solver).

Reply:
Additionally, I think the idea here is to also make the initial transform configurable in editor.